wear/watchface/build.gradle.kts

```diff
@@ -0,0 +1,44 @@
+/*
+ * Copyright 2025 The Android Open Source Project
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+plugins {
+    alias(libs.plugins.android.application)
+}
+
+android {
+    namespace = "com.android.developers.androidify.watchfacepush.defaultwf"
+    compileSdk = libs.versions.compileSdk.get().toInt()
+
+    defaultConfig {
+        applicationId = "com.android.developers.androidify.watchfacepush.defaultwf"
+        minSdk = 36
+        targetSdk = 36
+        versionCode = 1
+        versionName = "1.0"
```

Comment:
Does this version code etc need to be in sync with the other wearOS version code or not?